yarn-project/end-to-end/src/integration_l1_publisher.test.ts

```diff
@@ -267,12 +267,11 @@ describe('L1Publisher integration', () => {
         await makeBloatedProcessedTx(128 * i + 96),
         await makeBloatedProcessedTx(128 * i + 128),
       ];
-      // @todo @LHerskind fix time.
       const globalVariables = new GlobalVariables(
         new Fr(config.chainId),
         new Fr(config.version),
         new Fr(1 + i),
-        Fr.ZERO,
+        new Fr(await rollup.read.lastBlockTs()),
```

Comment:
Why do we want the ts of the last block here? Do we not want the current timestamp?

Reply:
The ts of the last block is the minimum value that will work. With the current timestamp, you have the issue of dealing with anvil not progressing time as "outside" so this was just an easy way to pick a meaningful value.